src/main/scala/encry/api/http/routes/WalletInfoApiRoute.scala

```diff
@@ -13,33 +14,37 @@ import encry.view.wallet.EncryWallet
 import io.circe.syntax._
 import org.encryfoundation.common.Algos
 import scala.concurrent.Future
+import encry.EncryApp.system
 
-case class WalletInfoApiRoute(nodeViewActorRef: ActorRef,
+case class WalletInfoApiRoute(nodeViewHolderRef: ActorRef,
                               restApiSettings: RESTApiSettings)(implicit val context: ActorRefFactory)
   extends EncryBaseApiRoute with FailFastCirceSupport {
 
-  override val route: Route = pathPrefix("wallet") { infoR ~ getUtxosR }
+  override val route: Route = pathPrefix("wallet") {
+    infoR ~ getUtxosR
+  }
 
   override val settings: RESTApiSettings = restApiSettings
 
-  private def getWallet: Future[EncryWallet] = (nodeViewActorRef ?
+  private def getWallet: Future[EncryWallet] = (nodeViewHolderRef ?
     GetDataFromCurrentView[EncryHistory, UtxoState, EncryWallet, EncryMempool, EncryWallet](_.vault))
     .mapTo[EncryWallet]
 
+  private def getBoxes: Future[Seq[EncryBaseBox]] =
+    (system.actorSelection("user/WalletStorageHolder") ? GetAllBoxes()).mapTo[Seq[EncryBaseBox]]
```

Comment:
You can make ref to WalletStorage a `val`, so you can replace `actorSelection` with it